smoke.sh

```diff
@@ -21,13 +21,26 @@ function cleanup()
     rm -rf /var/lib/glusterd /var/log/glusterfs/* /etc/glusterd $P/export;
 }
 
-function start_fs()
+function start_fs_with_arbiter()
 {
     mkdir -p $P/export;
     chmod 0755 $P/export;
 
     glusterd;
-    gluster --mode=script volume create $V replica 2 $H:$P/export/export{1,2,3,4} force;
+    gluster --mode=script volume create $V replica 3 arbiter 1 $H:$P/export/export{1,2,3,4,5,6} force;
```

Comment:
Any particular reason to choose arbiter over plain replica 3?

Reply:
I thought adding arbiter here will have more lines covered than plain replica 3.

Reply:
Ok, makes sense.